client/xdag_config.c

```diff
@@ -257,14 +256,11 @@ static char *xdag_config_read_node_name(FILE *fp)
 	return NULL;
 }
 
-
 bool isEmpty(const char *s)
```

Comment:
you are missing static :)

Reply:
Do you mean static modifier for isEmpty function?

Reply:
Fixed